src/Subscribers/SearchableSubscriber.php

```diff
@@ -72,10 +72,6 @@ public function getSubscribedEvents()
     public function postFlush(PostFlushEventArgs $args)
     {
         foreach ($this->indexable as $event) {
-            if (method_exists($event, 'isDeleted') && $event->isDeleted()) {
```

Comment:
I had this in my branch but removed it. Not sure why it appeared here. I still think this is needed in some form for soft deleteable-entities, otherwise they just get reindexed when using queued jobs.